aws-k8s-toolbox/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM debian:bullseye-slim
+LABEL REFRESHED_AT=20230606
+# image used inside Gitlab CICD Pipelines
+
+# Application list
+ARG KUBECTL_VERSION=1.25.9
+ARG HELM_VERSION=v3.9.0
+ARG KUBECONFORM_VERSION=0.4.14
+# env
+ENV BIN_DIR /opt/builder/bin
+ENV USER builder
+
+# OS depenencies
+RUN apt-get update && \
+    apt-get -y upgrade && \
+    apt-get install -y \
+       curl \
+       wget \
+       git \
+       apt-transport-https \
+       ca-certificates \
+       gnupg \
+       python3 \
+       python3-pip \
+       jq \
+       unzip \
+       make \
+       uuid-runtime \
+    && rm -rf /var/lib/apt/lists/*
+
+# workdir
+RUN useradd -ms /bin/bash ${USER} -d /opt/${USER}
+RUN mkdir -p ${BIN_DIR}
+WORKDIR ${BIN_DIR}
```

Comment:
FYI: when you use `WORKDIR` you don't need to create the fodler - it will be done automatically 👌

Reply:
Oops